### PR TITLE
diff for attachment fields: show link to old and new file

### PR DESCRIPTION
**Pull request by opqdonut**
for #1175 and #978

### CODE REVIEW

**Comment by opqdonut** [comment]
attachment-diff

**Comment by luontola** [comment]
It needs to handle the case of adding or removing an attachment:

liite

File: src/cljs/rems/fields.cljs [hunk 1]
```diff
@@ -205,19 +205,19 @@
...
-        filename-field [:div.field
-                        [:a.btn.btn-secondary.mr-2
-                         {:href (str "/api/applications/attachment/" value)
-                          :target :_new}
-                         filename " " (external-link)]]
+        link (fn [attachment-id filename]
+               [:div.field
+                [:a.btn.btn-secondary.mr-2
+                 {:href (str "/api/applications/attachment/" attachment-id)
+                  :target :_new}
```
Comment (luontola) [inline src/cljs/rems/fields.cljs]: Should be _blank https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target

**Comment by opqdonut** [comment]
attachment-diff-empty

**Pull request merged**